src/utilities/filterFunctions.js

```diff
@@ -0,0 +1,37 @@
+export function sortFun(state, products) {
+    const sortedProducts = products.sort((a, b) => a["price"] - b["price"]);
```

Comment:
for conversions to numbers, you can also append + before the entity you want to convert.

Reply:
Thanks, I have updated that.